packages/appstore/src/components/modals/wallet-modal/wallet-modal-body.tsx

```diff
@@ -47,7 +47,7 @@ const WalletModalBody = ({
                             is_bypassed={is_mobile}
                             is_scrollbar_hidden
                             height={content_heigth}
-                            width={max_content_width}
+                            style={{ maxWidth: max_content_width, width: '100%' }}
```

Comment:
This change has also been made [here](https://github.com/binary-com/deriv-app/pull/8794/files#diff-d4582e315d23c20c09d8a8acbf2aef88e829169cc94729bcd17e1f1734424bc9L50-L51) by @heorhi-deriv, it is needed for using `100%` instead of `100vw` [here](https://github.com/binary-com/deriv-app/pull/8753/files#diff-c522558c8707a0f5f8d1abeb72a922a7960586bcfde0a19eef2b36a776e3303cR6).